src/authentication/authmethods.js

```diff
@@ -0,0 +1,25 @@
+const jwt = require( 'jsonwebtoken' )
```

Comment:
Please do not use abbreviated terms in names - I suggest renaming this file `authentication.js`, or simply `index.js` if it's the only file in this directory